<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Load Parameter -->
  
  <param name="yolov5_path"       value="$(find yolov5_ros)/yolov5"/>
  <param name="use_cpu"           value="false" />

  <!-- Start yolov5 and ros wrapper -->
  <node pkg="yolov5_ros" type="yolo_v5_pc.py" name="yolov5_ros" output="screen" >
    <param name="weight_path"       value="$(find yolov5_ros)/weights/meters_gauges_ssdisplay.pt"/>
    <param name="image_topic"       value="/camera/color/image_raw" />
    <param name="depth_topic"       value="/camera/depth/image_rect_raw" />
    <param name="pub_topic"         value="/yolov5/BoundingBoxes" />
    <param name="camera_frame"       value="/camera/depth/camera_info"/>
    <param name="conf"              value="0.2" />
  </node>

</launch>
